src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb

```diff
@@ -48,7 +49,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             Inherits CompletionService
 
             Public Sub New(services As SolutionServices)
-                MyBase.New(services)
+                MyBase.New(services, AsynchronousOperationListenerProvider.NullProvider)
```

Comment:
This could get a bit confusing that this doesn't match what's actually in the MEF composition; even if it's not necessary, might be good to import the container's anyways just to keep it consistent.

Reply:
Done